src/pages/docs/packaging-applications/index.md

```diff
@@ -53,19 +53,19 @@ When you package your applications, you need to include all the binaries that ar
+| Package type    | File Extensions          | Notes                                                                                                                                                                                                                                                                                                                                                                                 |
+| --------------- | ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| NuGet           | .nupkg                   | Compatible with any NuGet repository (including the [Built-In repository](/docs/packaging-applications/package-repositories/built-in-repository)). Currently, only NuGet packages will have extra metadata like release notes and description extracted from the package metadata. Learn about NuGet on the [official NuGet website](http://docs.nuget.org/docs/start-here/overview). |
+| Zip             | .zip                     | Standard zip file as created through most common zip programs. Compatible with the [built-In repository](/docs/packaging-applications/package-repositories/built-in-repository) and external feeds.                                                                                                                                                                                   |
+| JAR WAR EAR RAR | .jar, .war, .ear, .rar   | Compatible with the [built-In repository](/docs/packaging-applications/package-repositories/built-in-repository/#pushing-packages-to-the-built-in-repository) and [Maven Feeds](/docs/packaging-applications/package-repositories/maven-feeds). RAR files are Java Resource Adaptor Archives, not the .rar compressed archive format.                                                 |
+| Tar             | .tar                     | Compatible with the [Built-In repository](/docs/packaging-applications/package-repositories/built-in-repository) only.                                                                                                                                                                                                                                                                |
+| Tar + Gzip      | .tgz, .tar.gz, .tar.Z    | Compatible with the [built-In repository](/docs/packaging-applications/package-repositories/built-in-repository) only.                                                                                                                                                                                                                                                                |
```

Comment:
Tested .tar.gz via external feed and this worked, so this section could also be updated.

Reply:
Also tested `.tgz` and `.tar.Z` - `.tgz` worked fine from an external feed, while `.tar.Z` did not (but worked via the built-in feed).